src/App.css

```diff
@@ -22,3 +22,20 @@
   from { transform: rotate(0deg); }
   to { transform: rotate(360deg); }
 }
+
+.coffee-list {
+  margin:10px;
+  padding: 10px;
+  width: 150px;
+  border: 2px solid gray;
+  background-color: #e3e3e3;
+  float: left;
+}
+
+.input-label {
+  color:green;
+}
+
+.coffee-size {
+  font-size: 85%;
+}
```

Comment:
this is the smallest of nits, but leave an extra line at the end of the file but only because it's "idiomatic" not for any other sensible reason.